cmd/codegen/generator/typescript/templates/src/testdata/type_test_enum_with_directive_want.ts

```diff
@@ -0,0 +1,49 @@
+
+/**
+ * Compression algorithm to use for image layers.
+ */
+export enum ImageLayerCompression {
+  EstarGz = "EStarGZ",
+  Estargz = "EStarGZ",
+  Gzip = "Gzip",
+  Uncompressed = "Uncompressed",
+  Zstd = "Zstd",
+}
+
+/**
+ * Utility function to convert a ImageLayerCompression value to its name so
+ * it can be uses as argument to call a exposed function.
+ */
+function ImageLayerCompressionValueToName(value: ImageLayerCompression): string {
+  switch (value) {
+    case ImageLayerCompression.EstarGz:
+      return "EStarGZ"
+    case ImageLayerCompression.Gzip:
+      return "Gzip"
+    case ImageLayerCompression.Uncompressed:
+      return "Uncompressed"
+    case ImageLayerCompression.Zstd:
+      return "Zstd"
+    default:
+      return value
+  }
+}
+
+/**
+ * Utility function to convert a ImageLayerCompression name to its value so
+ * it can be properly used inside the module runtime.
+ */
+function ImageLayerCompressionNameToValue(name: string): ImageLayerCompression {
+  switch (name) {
+    case "EStarGZ":
```

Comment:
Are we missing `Estargz` as a case both here and both? Seems like it *should* be here - we should also have a test that makes sure that *both* values work in queries.

Reply:
Can you add this as a test as I mentioned above? If we can do `dagger query` with both estargz values, then, SGTM as is, with no changes.